app/helpers/users_helper.rb

```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```

Comment:
What does this file do?

Reply:
I think this might have gotten created when I scaffolded (is that a verb?) the User model. Def don't need it for now!